epistemix_platform/src/epistemix_platform/app.py

```
@@ -26,6 +26,7 @@
 from epistemix_platform.controllers.job_controller import JobController
 from epistemix_platform.mappers.job_mapper import JobMapper
 from epistemix_platform.mappers.run_mapper import RunMapper
+from epistemix_platform.models.run import Run
```

Flake8 (7.3.0)

[error] 'epistemix_platform.models.run.Run' imported but unused

(F401)